packages/test/test-end-to-end-tests/package.json

```
@@ -40,7 +40,7 @@
 		"test:realsvc": "npm run test:realsvc:local && npm run test:realsvc:tinylicious",
 		"test:realsvc:frs": "npm run test:realsvc:run -- --driver=r11s --r11sEndpointName=frs --timeout=20s",
 		"test:realsvc:frs:report": "npm run test:realsvc:frs --",
-		"test:realsvc:local": "npm run test:realsvc:run -- --driver=local --timeout=3s",
+		"test:realsvc:local": "npm run test:realsvc:run -- --driver=local --timeout=15s",
```

Not a real suggestion, but to @alexvy86 's point about comments.... ;)
```suggestion
		"test:realsvc:local": "npm run test:realsvc:run -- --driver=local --timeout=15s && echo 'TEMP: Revert to 2s when AB#8942 is fixed",
```

I'm actually not a fan of these kinds of comments, I'd rather track the todo in the ADO item.  I mean, there's some value I guess, but I've seen enough 5-year-old `// TODO` comments to know it's not a good work tracking mechanism :)

---

I don't hate it 😄 . Totally agree this shouldn't be the main place for tracking, but as you said, I think it does add value to document why it's uncharacteristically high for people who happen upon it without knowing about the ADO item.